src/server.cpp

```diff
@@ -989,6 +1001,26 @@ JitterMeas.Measure();
     // one client is connected.
     if ( iNumClients > 0 )
     {
+
+        // Low frequency updates
+        if ( iFrameCount > CHANNEL_LEVEL_UPDATE_TIME )
+        {
+            iFrameCount = 0;
+
+            // Enable channel levels if any client has requested them
+            for ( int i = 0; i < iNumClients; i++ )
+            {
+                if ( vecChannels[ vecChanIDsCurConChan[i] ].ChannelLevelsRequired() )
+                {
+                    emit AudioFrames ( static_cast<int16_t>(iNumClients),
+                                       vecChanIDsCurConChan,
+                                       vecvecsData );
+                    break;
+                }
+            }
+        }
+        iFrameCount++;
```

Comment:
I would put this in an "else {}"

Reply:
Each frame causes the frame count to increment.  Putting it in an else would mean the first frame of each 200 did _not_ cause the count to increment, meaning 201 frames per update.

Reply:
Ah, ok. Thanks for the clarification.